server/src/test/java/org/opensearch/cluster/metadata/MetadataCreateIndexServiceTests.java

```diff
@@ -1349,18 +1355,20 @@ public void testClusterForceReplicationTypeInValidateIndexSettings() {
     }
 
     public void testRemoteStoreNoUserOverrideExceptReplicationTypeSegmentIndexSettings() {
+        ClusterState clusterState = ClusterState.builder(ClusterName.DEFAULT)
+            .nodes(DiscoveryNodes.builder().add(getRemoteNode()).build())
+            .build();
         Settings settings = Settings.builder()
             .put(CLUSTER_REPLICATION_TYPE_SETTING.getKey(), ReplicationType.DOCUMENT)
             .put(segmentRepositoryNameAttributeKey, "my-segment-repo-1")
```

Comment:
do we need this now? since cluster state is used to get repo values

Reply:
Need atleast one of segment or translog settings which are sought while checking `isRemoteDataAttributePresent(settings)`. Removing other superfluous settings.